nemo_skills/evaluation/metrics/translation_metrics.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import defaultdict
+
+from sacrebleu import corpus_bleu
+
+from nemo_skills.evaluation.metrics.base import BaseMetrics, as_float
+
+
+class TranslationMetrics(BaseMetrics):
```

Comment:
if you plan to refactor this, can you please add a todo and also submit a github issue, so that we can track this?